def ok():
    print ('OK!')

def exception():
    print ('ERROR!')
    raise ValueError
